addons/skin.estuary/xml/Home.xml

```diff
@@ -949,15 +949,23 @@
 						<item>
 							<label>$LOCALIZE[19020]</label>
 							<property name="menu_id">$NUMBER[12000]</property>
-							<onclick>ActivateWindow(TVGuide)</onclick>
+							<onclick condition="!Skin.String(pvr_tv_action)">Skin.SetString(pvr_tv_action,0)</onclick>
+							<onclick condition="!Skin.String(pvr_tv_action)">ActivateWindow(TVGuide)</onclick>
```

Comment:
Can this ever eval to true?  Doesn't the previous line get executed before this one?  Or is this a potential race?  ISTM this isn't needed (same thing for radio later)

Reply:
> Can this ever eval to true? Doesn't the previous line get executed before this one? Or is this a potential race? ISTM this isn't needed (same thing for radio later)

If the user doesn't go into setting first they both appear to be true.  If I don't set the 0, an empty value is created.  If I don't activate the window nothing popups unless I leave and comeback so the subsequent "true" test for 0 doesn't evaluate to zero.  I would be happy to know the proper way to initialize a default string.

Reply:
is the first condition not superfluous in that case? Why set the value if you check for an empty string anyways?

Reply:
I will try again, but if I go to settings afterwards I believe I either had trouble toggling to 1 when it is blank or displaying Guide for the default.

Reply:
So I noticed that I should initialize variables in the window onload for Home.xml and now the condition test work consistently in settings and from the menu.